### ISSUE TITLE
Testing limit sell

### ISSUE THREAD

**Issue by marinsokol5** [kroitor/ccxt#68]
Hi guys.
Using python and your latest version.

While trying to call HitBTC's method  `hitbtc.create_limit_sell_order`
This pops out.
```
Traceback (most recent call last):
  File "C:\Program Files (x86)\Python36-32\lib\site-packages\ccxt\__init__.py", line 249, in fetch
    response = opener.open (request, timeout = int (self.timeout / 1000))
  File "C:\Program Files (x86)\Python36-32\lib\urllib\request.py", line 532, in open
    response = meth(req, response)
  File "C:\Program Files (x86)\Python36-32\lib\urllib\request.py", line 642, in http_response
    'http', request, response, code, msg, hdrs)
  File "C:\Program Files (x86)\Python36-32\lib\urllib\request.py", line 570, in error
    return self._call_chain(*args)
  File "C:\Program Files (x86)\Python36-32\lib\urllib\request.py", line 504, in _call_chain
    result = func(*args)
  File "C:\Program Files (x86)\Python36-32\lib\urllib\request.py", line 650, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 409: Conflict

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "tests/test_ccxt.py", line 33, in f
    exchange.create_limit_sell_order('EOSETH', '0.0', '0.0')
  File "C:\Program Files (x86)\Python36-32\lib\site-packages\ccxt\__init__.py", line 593, in create_limit_sell_order
    return self.create_order (product, 'limit', 'sell', amount, price, params)
  File "C:\Program Files (x86)\Python36-32\lib\site-packages\ccxt\__init__.py", line 8279, in create_order
    return self.tradingPostNewOrder (self.extend (order, params))
  File "C:\Program Files (x86)\Python36-32\lib\site-packages\ccxt\__init__.py", line 8304, in request
    return self.fetch (url, method, headers, body)
  File "C:\Program Files (x86)\Python36-32\lib\site-packages\ccxt\__init__.py", line 284, in fetch
    self.raise_error (error, url, method, e, details)
  File "C:\Program Files (x86)\Python36-32\lib\site-packages\ccxt\__init__.py", line 222, in raise_error
    details,
TypeError: 'NoneType' object is not callable

```
As a product to be selled i have tried to put EOS/ETH and EOSETH but I am getting the same error.

------------------------------------------------------------------------------------------------------------
 On LiveCoin I get on the same method an error : 
```
Traceback (most recent call last):
  File "tests/test_ccxt.py", line 37, in f
    exchange.create_limit_sell_order('ETH/BTC', '0.0', '0.0')
  File "C:\Program Files (x86)\Python36-32\lib\site-packages\ccxt\__init__.py", line 593, in create_limit_sell_order
    return self.create_order (product, 'limit', 'sell', amount, price, params)
  File "C:\Program Files (x86)\Python36-32\lib\site-packages\ccxt\__init__.py", line 9384, in create_order
    return getattr (self, method) (self.extend (order, params))
AttributeError: 'livecoin' object has no attribute 'privatePostSelllimit'
```

------------------------------------------------------------------------------------------------------------
On Poloniex while calling the same method with 'ETH/BTC' didn't work but it did when i replaced it with 
BTC_ETH which is the way that their products are written.It would be amazing if we could call it with unified symbols.

Thank You and I am really sorry if one of these problems is coming from my side.

**Comment by kroitor** [comment]
Hello!

Let us do this step by step. Can you show some code you're using to initialize HitBTC? I mean, from creation of the instance up to a call to create_limit_sell_order? Sound like you forgot to load products. In order to test, please run this code:

```Python
import ccxt
hitbtc = ccxt.hitbtc ({
	"apiKey": "18339694544745d9357f9e7c0f7c41bb",
	"secret": "8340a60fb4e9fc73a169c26c7a7926f5",
})
# hitbtc.load_products ()
# hitbtc.fetch_balance ()
print (hitbtc.create_limit_sell_order ('EOS/ETH', 100.0, 100.0))
```

Can you please run this piece of code and copy its output here? After that, we will deal with Poloniex.

**Comment by kroitor** [comment]
Please, write if that snippet works, if that is the case, I will make an automatic product preloading, so that nobody ever forgets to load them )

**Comment by marinsokol5** [comment]
Yeah,that was the problem with HitBTC.My bad.
Poloniex and livecoin stil broken unfortunately.

**Comment by kroitor** [comment]
The Livecoin issue is a bug in the library, indeed, thanks for posting this! I have uploaded the fix for it and it should be working within 20-30 minutes in the next version 1.1.127. That is the Livecoin part. So, I consider it fixed, thanks again.
Now I am going to look deeper into Poloniex.

**Comment by kroitor** [comment]
Code for testing poloniex:
```
import ccxt
poloniex = ccxt.poloniex ({
	"apiKey": "NAC75MYB-R2G5KRO5-IVBTG5RQ-VWR05TFJ",
	"secret": "e4d27ab551a565e606b67a50067b629bdfda62b61e33a4a5f98f02ed2d4f29c68edf0e49487a18adfb83833fd59ef99d14e8abb66f2657177f00dd941ff99d07",
	"verbose": True,
})
poloniex.load_products ()
print (poloniex.create_limit_sell_order ('ETH/BTC', 100.0, 100.0))
```

What output do you get if you run this snippet?

**Comment by kroitor** [comment]
With poloniex the problem is probably the same as with HitBTC. Should work, if you loaded products previously. I will definitely make a silent auto-preload in the next version for this issue to go away forever )

**Comment by kroitor** [comment]
Alright, livecoin should be available now since 1.1.127 and should work as expected. Please update your version, then try making a real or fake livecoin order and paste the output here (with verbose mode = True). The issue with Livecoin should be gone.

**Comment by marinsokol5** [comment]
Yeah they are all gone,thank You very much.
